R/bambu-processReads_scoreReadClasses.R

```diff
@@ -123,21 +123,16 @@ getGeneScore = function(rowData){
 #' calculates the minimum FDR for each score 
 calculateFDR = function(score, labels){
   scoreOrder = order(score, decreasing = T)
-  orderSave = (1:length(score))[scoreOrder]
   labels = labels[scoreOrder]
   score = score[scoreOrder]
-  FDR = cumsum(labels)/(1:length(score))
-  FDR = rev(cummin(rev(FDR)))
```

Comment:
Can't remove this line in the case the FDR below an expected threshold
Below isn't the clearest example but its shows how the line smooths peaks so that any value to the left of another will always be smaller.
![image](https://user-images.githubusercontent.com/17838991/113647124-ce0e6500-96bc-11eb-8e9e-ddfec1e59d4c.png)
![image](https://user-images.githubusercontent.com/17838991/113647150-dcf51780-96bc-11eb-8fe7-12a508f4e94c.png)

Reply:
I put it back in in 8537f13